### ISSUE TITLE
On pickup not working correctly

### ISSUE THREAD

**Issue by Griffeng** [Matocolotoe/Skript-1.8#54]
### Skript Version

[21:07:19 INFO]: [Skript] Skript's aliases can be found here: https://github.com/SkriptLang/skript-aliases
[21:07:19 INFO]: [Skript] Skript's documentation can be found here: https://skriptlang.github.io/Skript
[21:07:19 INFO]: [Skript] Server Version: git-PaperSpigot-445 (MC: 1.8.8)
[21:07:19 INFO]: [Skript] Skript Version: 2.6.1
[21:07:19 INFO]: [Skript] Installed Skript Addons: 
[21:07:19 INFO]: [Skript]  - DiSky v2.1 (https://disky.itsthesky.info/)
[21:07:19 INFO]: [Skript]  - skript-reflect v2.3 (https://github.com/TPGamesNL/skript-reflect)
[21:07:19 INFO]: [Skript]  - skRayFall v1.9.20 (https://sk.rayfall.net/)
[21:07:19 INFO]: [Skript]  - SkBee v1.10.2 (https://github.com/ShaneBeee/SkBee)
[21:07:19 INFO]: [Skript] Installed dependencies: 
[21:07:19 INFO]: [Skript]  - SQLibrary v7.1
[21:07:19 INFO]: [Skript]  - Vault v1.7.2-b107
[21:07:19 INFO]: [Skript]  - WorldGuard v6.1

### Bug Description

This is the error, when using the on pickup event

```
[20:59:54 ERROR]: Could not pass event PlayerPickupItemEvent to Skript v2.6.1
java.lang.NoClassDefFoundError: org/bukkit/event/entity/EntityPickupItemEvent
        at ch.njol.skript.events.EvtItem.check(EvtItem.java:155) ~[?:?]
        at ch.njol.skript.SkriptEventHandler.check(SkriptEventHandler.java:136) ~[?:?]
        at ch.njol.skript.SkriptEventHandler.access$000(SkriptEventHandler.java:53) ~[?:?]
        at ch.njol.skript.SkriptEventHandler$PriorityListener.lambda$new$0(SkriptEventHandler.java:71) ~[?:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78) ~[patched_1.8.8.jar:git-PaperSpigot-445]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.8.8.jar:git-PaperSpigot-445]
        at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:517) [patched_1.8.8.jar:git-PaperSpigot-445]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:502) [patched_1.8.8.jar:git-PaperSpigot-445]
        at net.minecraft.server.v1_8_R3.EntityItem.d(EntityItem.java:314) [patched_1.8.8.jar:git-PaperSpigot-445]
        at net.minecraft.server.v1_8_R3.EntityHuman.d(EntityHuman.java:449) [patched_1.8.8.jar:git-PaperSpigot-445]
        at net.minecraft.server.v1_8_R3.EntityHuman.m(EntityHuman.java:440) [patched_1.8.8.jar:git-PaperSpigot-445]
        at net.minecraft.server.v1_8_R3.EntityLiving.t_(EntityLiving.java:1495) [patched_1.8.8.jar:git-PaperSpigot-445]
        at net.minecraft.server.v1_8_R3.EntityHuman.t_(EntityHuman.java:174) [patched_1.8.8.jar:git-PaperSpigot-445]
        at net.minecraft.server.v1_8_R3.EntityPlayer.l(EntityPlayer.java:293) [patched_1.8.8.jar:git-PaperSpigot-445]
        at net.minecraft.server.v1_8_R3.PlayerConnection.a(PlayerConnection.java:385) [patched_1.8.8.jar:git-PaperSpigot-445]
        at net.minecraft.server.v1_8_R3.PacketPlayInFlying.a(SourceFile:126) [patched_1.8.8.jar:git-PaperSpigot-445]
        at net.minecraft.server.v1_8_R3.PacketPlayInFlying$PacketPlayInPositionLook.a(SourceFile:18) [patched_1.8.8.jar:git-PaperSpigot-445]
        at net.minecraft.server.v1_8_R3.PlayerConnectionUtils$1.run(SourceFile:13) [patched_1.8.8.jar:git-PaperSpigot-445]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
        at net.minecraft.server.v1_8_R3.SystemUtils.a(SourceFile:44) [patched_1.8.8.jar:git-PaperSpigot-445]
        at net.minecraft.server.v1_8_R3.MinecraftServer.B(MinecraftServer.java:774) [patched_1.8.8.jar:git-PaperSpigot-445]
        at net.minecraft.server.v1_8_R3.DedicatedServer.B(DedicatedServer.java:378) [patched_1.8.8.jar:git-PaperSpigot-445]
        at net.minecraft.server.v1_8_R3.MinecraftServer.A(MinecraftServer.java:713) [patched_1.8.8.jar:git-PaperSpigot-445]
        at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:616) [patched_1.8.8.jar:git-PaperSpigot-445]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
```

### Expected Behavior

On pickup should work and not spit out error

### Steps to Reproduce

Use a pickup event and pickup any item

### Agreement

- [X] I have read the guidelines above and confirm I am following them with this report.

**Comment by Matocolotoe** [comment]
Already fixed in https://github.com/Matocolotoe/Skript-1.8/commit/bd32f1ee1d40f012c84cea967a8af200f731f7c3 and will be included in next release.

**Comment by Griffeng** [comment]
> Already fixed in [bd32f1e](https://github.com/Matocolotoe/Skript-1.8/commit/bd32f1ee1d40f012c84cea967a8af200f731f7c3) and will be included in next release.

Ohh yeah diden't see that, sorry.
